packages/nodes-base/nodes/Spotify/GenericFunctions.ts

```diff
@@ -0,0 +1,80 @@
+import { OptionsWithUri } from 'request';
+
+import {
+	IExecuteFunctions,
+	IHookFunctions,
+} from 'n8n-core';
+
+/**
+ * Make an API request to Spotify
+ *
+ * @param {IHookFunctions} this
+ * @param {string} method
+ * @param {string} url
+ * @param {object} body
+ * @returns {Promise<any>}
+ */
+export async function spotifyApiRequest(this: IHookFunctions | IExecuteFunctions, method: string, endpoint: string, body: object, query?: object): Promise<any> { // tslint:disable-line:no-any
```

Comment:
Would be better to solve this `tslint` issue globally or fix the error.

Reply:
https://palantir.github.io/tslint/usage/configuration/

Reply:
I am not seeing a tslint error on this line, though I did fix the tslint errors in other parts of the code since this comment.

Reply:
He means the comment `// tslint:disable-line:no-any`, which makes an exception for `Promise<any>`. This happens very often in the codebase. I'll be looking into whether we can configure TSLint to do away with these exceptions.